generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -1,21 +1,19 @@
-<%_ if (integrations.includes('docker')) { _%>
```

Comment:
On official gitlab, Docker is used.
On the private gitlab on my current project at work, it used workers without Docker.
So I added back the condition, because there is a question with/without Docker